packages/yoastseo/src/scoring/assessments/seo/ProductIdentifiersAssessment.js

```diff
@@ -53,6 +53,11 @@ export default class ProductIdentifiersAssessment extends Assessment {
 			assessmentResult.setText( result.text );
 		}
 
+		if ( assessmentResult.getScore() < 9  ) {
+			assessmentResult.setHasJumps( true );
+			assessmentResult.setEditFieldName( __( "Product identifiers", "wordpress-seo" ) );
```

Comment:
I am not sure `"wordpress-seo"` domain is the right one here. The assessment is not bundled in Free and only loaded in `wpseo-woocommerce`. However, the edit button is only shown in Premium.

Reply:
Per this discussion, the domain should be the WooCommerce domain, so this needs to be changed to `"yoast-woo-seo"`